source/reference/api/reverse.txt

```diff
@@ -0,0 +1,79 @@
+.. _c2c-api-reverse:
+
+===========
+``reverse``
+===========
+
+.. include:: /includes/preview-warning
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Description
+-----------
+
+Reverses the direction of a committed sync operation.
+
+For example, if you have committed a sync operation where ``cluster0``
+is the source and ``cluster1`` is the destination, if you use the
```

Comment:
if you have committed a sync operation where cluster0 is the source and cluster1 is the destination, and only new writes occur on the destination cluster, then you can use the reverse endpoint to sync writes from cluster1 to cluster0